docs/source/concepts/ledger-topologies.rst

```diff
@@ -0,0 +1,179 @@
+   :align: center
+
+DAML on `VMware Concord <https://blogs.vmware.com/blockchain>`__ and DAML on `Hyperledger Sawtooth <https://sawtooth.hyperledger.org/>`__ are examples of such a replicated setup.
+
+Trusted Execution Environments
+==============================
+
+Integrity and privacy can also be protected using hardware Trusted Execution Environments (TEEs), such as Intel SGX.
+The software implementing the ledger can then be deployed inside of TEE "enclaves", which are code blocks that the processor isolates and protects from the rest of the software stack (even the operating system).
+The hardware ensures that the enclave data never leaves the processor unencrypted, offering privacy.
+Furthermore, hardware-based attestation can guarantee that the operating entities process data using the prescribed code only, guaranteeing integrity.
+The hardware is designed in such a way as to make any potential physical attacks by the operating entity extremely expensive.
+This moves the trust necessary to achieve these properties from the operators of the trust domains that maintain the global state to the hardware manufacturer, who is anyway trusted with correctly producing the hardware.
+Recent security research has, however, found scenarios where the TEE protection mechanisms can be compromised.
+
+.. _partitioned-topologies:
+
+Partitioned Ledger Topologies
+*****************************
+
+In these topologies, the ledger is implemented as a distributed system.
+Unlike the global state topologies, no single trust domain holds a physical copy of the entire shared ledger.
+Instead, the participant nodes hold just the part of the ledger (i.e., the :ref:`ledger projection <da-model-projections>`) that is relevant to the parties to whom they serve the Ledger API.
+The participants jointly extend the ledger by running a distributed commit protocol.
+
+.. image:: ./images/ledger-topologies/decentralized-ledger-topology.svg
+   :align: center
+
+The implementations might still rely on trusted third parties to facilitate the commit protocol.
+The required trust in terms of privacy and integrity, however, can generally be lower than in global state topologies.
+Moreover, unlike the previous topologies, they support interoperability: even if two transactions are committed with the help of disjoint sets of trusted third parties, their :ref:`output contracts <def-output-contract>` can in general still be used within the same atomic transaction.
+The exact trust assumptions and the degree of supported interoperability are implementation-dependent.
+`Canton <http://canton.io>`__ and DAML on `R3 Corda <https://www.corda.net>`__ are two such implementations.
+The main drawback of this topology is that availability can be influenced by the participant nodes.
+In particular, transactions cannot be committed if they use data that is only stored on unresponsive nodes.
+Spreading the data among additional trusted entities can mitigate the problem.
```

Comment:
Completed review of topology section: very nice decomposition of topologies @oggy- ! This is going to be a good basis for providing a "Which DAML ledger should I use?" section!